lumicks/pylake/correlated_stack.py

```diff
@@ -115,6 +115,23 @@ def crop_by_pixels(self, x_min, x_max, y_min, y_max):
         data = self.src.with_roi(np.array([x_min, x_max, y_min, y_max]))
         return self.from_dataset(data, self.name, self.start_idx, self.stop_idx)
 
+    def get_image(self, channel="rgb"):
+        """Get image data for the full stack as an `np.ndarray`.
+
+        Parameters
+        ----------
+        channel : {'red', 'green', 'blue', 'rgb'}
+            The color channel of the requested data.
+            For single-color data, this argument is ignored.
+        """
+        if self.src._description.is_rgb:
+            channel_indices = {"red": 0, "green": 1, "blue": 2, "rgb": slice(None)}
+            slc = (slice(None), slice(None), channel_indices[channel])
+        else:
+            slc = (slice(None),)
+
+        return np.stack([frame.data[slc] for frame in self], axis=0)
```

Comment:
Correct me if I'm wrong, but I think you need to squeeze this one to get rid of the extraenous axis (if you want to make it fully consistent with `Scan`)?

I found it when trying to slice the stack. Might also be good to add a test for this:
```
np.testing.assert_array_equal(np.squeeze(stack[0].get_image()), rgb_data)
```